app/Database/Entities/Account.php

```diff
@@ -0,0 +1,80 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Database\Entities;
+
+use App\Database\Schema\AccountSchema;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity()
+ */
+class Account extends AbstractEntity
+{
+    use AccountSchema;
+
+    /**
+     * Constant variable for account number rule.
+     *
+     * @const string
+     */
+    public const ACCOUNT_NUMBER_RULE = 'required|string';
+
+    /**
+     * Constant variable for lifetime.
+     *
+     * @const string
+     */
+    public const SUBSCRIPTION_TYPE_LIFETIME = 'lifetime';
+
+    /**
+     * Constant variable for monthly.
+     *
+     * @const string
+     */
+    public const SUBSCRIPTION_TYPE_MONTHLY = 'monthly';
```

Comment:
Kindly add SUBSCRIPTION_TYPES as well which contains all subscription types in an array